spec/features/dashboard/collection_spec.rb

```diff
@@ -73,38 +73,79 @@
   end
 
   describe 'create collection' do
-    before do
-      collection_type
-      user_collection_type
+    let(:title) { "Test Collection" }
+    let(:description) { "Description for collection we are testing." }
 
-      sign_in user
-      visit '/dashboard/my/collections'
+    context 'when user can create collections of multiple types' do
+      before do
+        collection_type
+        user_collection_type
+
+        sign_in user
+        visit '/dashboard/my/collections'
+      end
+
+      it "makes a new collection", :js do
+        click_button "New Collection"
+        expect(page).to have_content 'Select type of collection'
+
+        choose('User Collection')
+        click_on('Create collection')
+
+        expect(page).to have_content 'Create New Collection'
+        expect(page).to have_selector "input.collection_title.multi_value"
+
+        click_link('Additional fields')
+        expect(page).to have_selector "input.collection_creator.multi_value"
+
+        fill_in('Title', with: title)
+        fill_in('Abstract or Summary', with: description)
+        fill_in('Related URL', with: 'http://example.com/')
+
+        click_button("Create Collection")
+        expect(page).to have_content 'Items'
+        expect(page).to have_content title
+        expect(page).to have_content description
+      end
     end
 
-    let(:title) { "Test Collection" }
-    let(:description) { "Description for collection we are testing." }
+    context 'when user can create collections of one type' do
+      before do
+        user_collection_type
 
-    it "makes a new collection", :js do
-      click_button "New Collection"
-      expect(page).to have_content 'Select type of collection'
+        sign_in user
+        visit '/dashboard/my/collections'
+      end
 
-      choose('User Collection')
-      click_on('Create collection')
+      it 'makes a new collection' do
+        click_link "New Collection"
```

Comment:
@jeremyf I'm a little concerned that this test is twitchy.  [Hyrax::Dashboard::CollectionsController#new](https://github.com/samvera/hyrax/blob/collections-sprint/app/controllers/hyrax/dashboard/collections_controller.rb#L70) method seems to fail randomly at line `@collection.apply_depositor_metadata(current_user.user_key)`.  The test just passed in Travis for rails 5.0, but I had to re-run it once for rails 5.1.  I don't think it is the rails version.  An earlier commit had it fail on both.

Reply:
For what it is worth, it passes on my local machine every time.

Reply:
@elrayle I have time after 2pm today to review and look at possible solutions.